src/broker/cas_execute.c

```diff
@@ -1064,7 +1062,7 @@ ux_cgw_prepare (char *sql_stmt, int flag, char auto_commit_mode, T_NET_BUF * net
       goto prepare_error;
     }
 
-  err_code = cgw_get_handle (&srv_handle->cgw_handle, true);
+  err_code = cgw_get_handle (&srv_handle->cgw_handle);
```

Comment:
no necessary to check cgw_handle is null?

Reply:
The cgw_get_handle function allocates a cgw handle to srv_handle->cgw_handle.
And if cgw_handle is null, it returns an error.